rst/dev-guide/_deconst.json

```diff
@@ -1,3 +1,3 @@
 {
-  "contentIDBase": "https://github.com/rackerlabs/docs-cloud-load-balancers/rst/dev-guide"
+  "contentIDBase": "https://github.com/rackerlabs/docs-cloud-load-balancers/"
```

Comment:
Technically you can use any string at all for the ID base, as long as it's unique, but this makes it consistent with all of the other content IDs, and keeps our content mapping in the control repository from looking weird.